serveradmin/api/migrations/0001_api_lock.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.15 on 2020-11-02 13:15
```

Comment:
Uhm, we now have two identical migrations here.

Reply:
Fxied